include/belle.hh

```diff
@@ -1196,10 +1196,15 @@ private:
 
   private:
 
+#ifndef _MSC_VER
```

Comment:
Since the 'if' has an 'else', instead of testing for false, could it test for true and swap the bodies around:
```cpp
#ifdef _MSC_VER
...
#else
...
#endif 
```

Reply:
I'd liked the idea of having the real/origin code above the workaround. But if you prefer it the other way that's no problem.